pkg/util/json/json.go

```diff
@@ -231,22 +230,64 @@ func (j jsonNumber) String() string { return asString(j) }
 func (j jsonArray) String() string  { return asString(j) }
 func (j jsonObject) String() string { return asString(j) }
 
+const hexAlphabet = "0123456789abcdef"
+
 // encodeJSONString writes a string literal to buf as a JSON string.
-// Very similar to encodeSQLStringInsideArray. Primary difference is that it is
-// legal to directly print out unicode characters.
-func encodeJSONString(buf *bytes.Buffer, in string) {
+// Cribbed from https://github.com/golang/go/blob/master/src/encoding/json/encode.go.
```

Comment:
Can you make this a permalink? Here and below. (Pressing <kbd>y</kdb> on the GH page will permalinkize the URL.)

Reply:
I was not, that's useful!